pkg/cli/cmd/app/connections/display_test.go

```diff
@@ -104,6 +108,25 @@ func Test_display(t *testing.T) {
 					},
 				},
 			},
+			{
+				ID:                &azureRedisID,
+				Name:              &azureRedisName,
+				Type:              &azureRedisType,
+				ProvisioningState: &provisioningStateSuccess,
+				OutputResources: []*corerpv20231001preview.ApplicationGraphOutputResource{
+					{
+						ID:   &azureRedisID,
+						Name: &azureRedisName,
+						Type: &azureRedisType,
+					},
+				},
+				Connections: []*corerpv20231001preview.ApplicationGraphConnection{
+					{
+						Direction: &dirInbound,
+						ID:        &sqlDbID,
+					},
+				},
+			},
 		}
 
 		expected := `Displaying application: test-app
```

Comment:
I was thinking about using "html/template" and a tml file. This "expected" String is a little ugly but maybe simpler?

Reply:
You could also use a plain text file (eg: `testdata/expected-display-output.txt`). Do we need templating? It looks like the expected output is fully static.

Reply:
I dislike the string concatenation, I don't want a file IO and templates are pointless (here).... hmm

* Current - concatenation and mixing string operations " ` is ugly
* Templates - As this is static there is no point
* File - The overhead of IO ops is excessive - unpresidented within the code base

I will have a think today...

Reply:
By the way we use files in tests for expected output all the time. File IO definitely isn't a concern for me in tests.